### DISCUSSION TITLE
Move RXjava and RXjava2 to legacy

### DISCUSSION THREAD

**Discussion by rajadileepkolli** [hantsy/spring-reactive-sample#171] [Ideas]
As per spring boot [issue](https://github.com/spring-projects/spring-framework/issues/27443) , spring will stop supporting RXjava and RXJava2 , So I propose to move these projects to legacy, if you are inclined with this I can raise a PR

**Comment by hantsy** [comment]
OK, create a PR for it please.

**Reply by rajadileepkolli** [reply]
Raised PR #172 for the same
